gems/sorbet/lib/require_everything.rb

```diff
@@ -106,6 +104,41 @@ def self.patch_kernel
 
   private
 
+  def self.rb_file_paths
+    srb_path = __FILE__
+    matcher = %r{^(.*/gems/)sorbet-([.\d]*)/*}
+    match = matcher.match(srb_path)
+
+    unless match
+      raise "Could not match with sorbet at #{srb_path}"
+    end
+
+    gem_path = match[1]
+    version_number = match[2]
+
+    glob = gem_path + "/sorbet-static-#{version_number}*/libexec/sorbet"
+    unless Dir[glob].any?
+      raise "Could not find sorbet-static executable for sorbet at #{srb_path}"
+    end
+
+    # Should be only one of these per-version
+    sorbet_exec = Dir[glob].first
+    output = `#{sorbet_exec} -p file-table-json --stop-after=parser`
```

Comment:
This would be a good place to pipe in uses of the `--typed-override`

Reply:
Good call. Do you think it makes sense to pass through all additional command line args?

Reply:
When Sorbet is started, it will read `sorbet/config`. The config file is where any `--typed-override` flags should live (we don't currently have any real arg parsing for the RBI generation subcommands of `srb`, and it'd be nice to keep it that way, especially if the options we're considering adding are just duplicated with `sorbet-static`.

Reply:
Good point! That's a lot less to plumb through this PR as well :)